lib/cusparse/generic.jl

```diff
@@ -216,6 +217,11 @@ function mm!(transa::SparseChar, transb::SparseChar, alpha::Number, A::Union{CuS
         return out[]
     end
     with_workspace(bufferSize) do buffer
+        # Uncomment is we find a way to reuse the buffer
```

Comment:
Typo. Also, could you elaborate a little more, and/or link to an issue? Is https://github.com/JuliaGPU/CUDA.jl/issues/1362 related?

Reply:
Yes, it's exactly the issue #1362!
I didn't know that @frapac already opened an issue about it.